lib/screens/play/card/card_pile.dart

```diff
@@ -1,33 +1,48 @@
 import 'package:flutter/material.dart';
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
```

Comment:
Would be good to add some tests for this - there are none currently. Especially now that it has some logic, we can test that e.g:

 - Clicking discard when it's empty does nothing
 - Clicking discard when non-empty has the expected effect on the game state
 - The 'deck' pile doesn't render as a button, and has the correct size rendered